lib/fhir_models/definitions/structures/profiles-types.json

```diff
@@ -2844,7 +2844,7 @@
                 "ele-1"
               ],
               "constraint": [
-               {
+                {
```

Comment:
We don't want to apply rubocop to the definitions, since they come from HL7 and maintaining them doesn't make sense. We only update them when FHIR makes an official release.